middle_end/flambda/types/env/typing_env.rec.ml

```diff
@@ -732,33 +764,40 @@ let binding_time_and_mode_of_simple t simple =
         Binding_time.consts_and_discriminants Name_mode.normal)
     ~name:(fun name -> binding_time_and_mode t name)
 
+let mem_by ?min_name_mode t name ~find ~in_imported_names =
```

Comment:
This should be `[@inline always]` otherwise closures might be allocated at the call sites (for the arguments).